widget/table.go

```diff
@@ -541,6 +552,33 @@ func (t *Table) ScrollToTop() {
 	t.finishScroll()
 }
 
+// scrollByOnePage scrolls down or up by table height
+func (t *Table) scrollByOnePage(down bool) {
+	if t.Length == nil || t.content == nil {
```

Comment:
You may need to add some offset validation here - please test what happens if you try to keep scrolling when you're already at the top or bottom of the list

Reply:
There is one bug in the tree widget, but its not introduced with this PR, it just became obvious after implementing it.

When you call tree.ScrollToBottom() twice, the last row's top border vanishes.

Other than that, there are no problems. Not sure if I should investigate it, since its not part of this PR.

I can share some example code (120 lines).

Reply:
If its not related then open an issue - the example code will be useful there :)